packages/qemu.rb

```diff
@@ -32,9 +26,13 @@ class Qemu < Package
   depends_on 'libusb'
   depends_on 'lzo'
   depends_on 'pixman'
+  depends_on 'hicolor_icon_theme'
```

Comment:
Why do we need this?

Reply:
OK, fair enough.  What was the full command line you used to test including all arguments?

Reply:
`qemu-system-i386`? I double checked that it started and that it didn't give me the `***stack smashing detected***` error. I can more thoroughly test, if you'd like.

Reply:
Yeah, but if you run `qemu-system-i386` with no options, it obviously won't start a session.  Do you understand what I'm asking here?

Reply:
It compiles, runs, and the GUI pops up. That's as far as I tested.

Reply:
Here:
```
wget http://nopid.free.fr/small.ffs.bz2
bunzip2 small.ffs.bz2
qemu-system-i386 -hda small.ffs 
```
This is tested and working on x86_64.

Reply:
Is that an option template?  Looks useful.  Where do you find these recipes?

Reply:
What? No, these are the commands that I used to test it. I got the URL from https://wiki.qemu.org/Testing/System_Images and from there I just knew the rest.

Reply:
Oh, that is not exactly what I expected.  Probably want to test with something like this:
```
qemu-system-i386 -hda R60-9592.96.2017_09_20_2221_qemu.img
```
This requires about 5 gb of free disk space, however.

Reply:
No way. What I used was good enough. That never worked anyway, you know. Even with the old version of QEMU. Please stop dreaming.

Reply:
No way. What I used was good enough. That never worked anyway, you know. Even with the old version of QEMU.

Reply:
I have to use VirtualBox on my Debian laptop for that image to work. Actually, the Cloudready image works directly on her laptop with no VM, so that's what I use.

Reply:
I assume her laptop is your mother's?  She has a lot of faith in you to let you mess with her system. :)

Reply:
She doesn't care. All she does is watch movies on it. I run VirtualBox in Linux now, because Windows isn't working.